.github/workflows/test_objectivec.yml

```diff
@@ -68,17 +68,23 @@ jobs:
     strategy:
       fail-fast: false   # Don't cancel all jobs if one fails.
       matrix:
-        PLATFORM: ["ios", "macos", "tvos", "watchos"]
+        PLATFORM: ["ios", "macos", "tvos", "watchos", "visionos"]
         CONFIGURATION: ["Debug", "Release"]
-    name: CocoaPods ${{ matrix.PLATFORM}} ${{ matrix.CONFIGURATION}}
-    runs-on: macos-12
-    env:
-      DEVELOPER_DIR: /Applications/Xcode_14.1.app/Contents/Developer
+        include:
+          - OS: macos-12
+          - OS: macos-14
```

Comment:
Can we bump this down to macos-13?  It looks like that supports the xcode version you need, and isn't in beta

Reply:
When I run pod lib lint on my own repo based on macos-13 image and Xcode 15.2，the error's description as [visionOS] unknown: Encountered an unknown error (Could not find a xros simulator (valid values: ios, tvos, watchos). appeared also.
See: https://github.com/growingio/growingio-sdk-ios-utilities/actions/runs/7984766703/job/21802111821
And: https://github.com/actions/runner-images/issues/8144#issuecomment-1810765004